controller/api/destination/server.go

```diff
@@ -215,6 +216,8 @@ func (s *server) GetProfile(dest *pb.GetDestination, stream pb.Destination_GetPr
 		path = dest.GetPath()
 	}
 
+	translator = newProfileTranslator(stream, log, &service, s.clusterDomain)
```

Comment:
I think you should just be able to use `:=` here and not declare translator above.

The `translator` variable that you declare in the inner scope on line 199 is already out of scope here.